### ISSUE TITLE
Validate and test mysql backup strategy

### ISSUE THREAD

**Issue by pmbrull** [open-metadata/OpenMetadata#6343]
We had some issues trying to backup and restore a MySQL db with OM data. Some notes:

---

1. The original issue when trying to restore the data was caused due to `mysqldump` not properly handling GENERATED columns.
2. We updated the SQL dumps with the following commands to patch generated columns:
    `sed -i 's/GENERATED ALWAYS AS .* VIRTUAL//' mydump.sql`
    `sed -i 's/GENERATED ALWAYS AS .* STORED//' mydump.sql`
3. Afterward, the restore ran properly on `openmetadata_db`.

Next, a pending action is still required to clean up the database state fully:
 - As we patched some generated columns, we need to ALTER the tables to return them to the desired state. This means dropping the patched columns (which should only contain generated data) and recreating them with the right GENERATED ALWAYS AS statement. E.g.,

```
mysql -e "ALTER TABLE foo DROP COLUMN bar;\
ALTER TABLE foo ADD COLUMN bar VARCHAR(255) AS ...;"
```

---

We should validate the whole backup process, see if we can replicate this issue, and if we can, prepare some backup utilities to help users run this process easily instead of having to patch the whole `mysqldump` utility

**Comment by pmbrull** [comment]
Let's use the following PR for the backup https://github.com/open-metadata/OpenMetadata/pull/6488

**Comment by harshach** [comment]
@pmbrull @akash-jain-10 anything remaining here?

**Comment by akash-jain-10** [comment]
Hello @harshach . This is fixed with 0.11.5 Release. We can mark this as closed.
